tests/coverage/test_gcovr.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+##############################################################
+# Folders structure (28.07.2020)
+# NOTE: Drawing will be removed after approval script
+##############################################################
+# facelift/
+# ├─ build/
+# │  ├─
+# │  └─
+# ├─ ci/
+# │  ├─ appveyor
+# │  └─ travis
+# ├─ cmake/
+# ├─ CMakeLists.txt
+# ├─ src/
+# ├─ tests/
+# │  ├─ asyncfunctions/
+# │  ├─ benchmarking/
+# │  ├─ cmake/
+# │  ├─ CMakeLists.txt
+# │  ├─ combined/
+# │  ├─ coverage/
+# │  │   ├─ read.me
+# │  │   └─ test_gcov.sh*
+
+DIR_CVRG=$(cd "$(dirname "$0")"; pwd -P)
+DIR_TESTS=$(cd "$(dirname "$DIR_CVRG")"; pwd -P)
+DIR_SRC=$(cd "$(dirname "$DIR_TESTS")"; pwd -P)
+BUILD_DIR=$PWD
+DATE_TIME=$(date +"%y-%m-%d-%H%M%S")
+DIR_REPORT="$BUILD_DIR/gcovr/report_gcovr_${DATE_TIME}"
+DIR_CI="$DIR_SRC/ci/travis"
+DIR_GTEST="$BUILD_DIR/gtest"
+DIR_GTEST_INS="$DIR_GTEST/googletest/install"
```

Comment:
INS ???
Make our scripts readable please.

Reply:
INS is reduce form of the INSTALL. If you don't mind then will be DIR_GTEST_INSTALL